catalyst-gateway/tests/api_tests/utils/rbac_chain.py

```diff
@@ -54,29 +67,38 @@ def cat_id_for_role(self, role_id: RoleID) -> (str, str):
                 is_uri=True,
             ),
             role_data_arr[-1]["sk"],
-        )    
-    
-    def short_cat_id(self) -> str:        
+        )
+
+    def short_cat_id(self) -> str:
         return generate_cat_id(
             network=self.network,
             subnet=self.subnet,
             pk_hex=self.keys_map[f"{RoleID.ROLE_0}"][0]["pk"],
             is_uri=False,
         )
 
+
 @pytest.fixture
 def rbac_chain_factory():
-    def __rbac_chain_factory(role_id: RoleID, network: str = "cardano", subnet: str = "preprod") -> RBACChain:
-        match role_id:
+    def __rbac_chain_factory(
+        chain: Chain = Chain.All,
+        network: str = "cardano",
+        subnet: str = "preprod",
+    ) -> RBACChain:
+        match chain:
             # RBAC registration chain that contains only Role 0 (voter)
-            case RoleID.ROLE_0:
+            case Chain.Role0:
                 return RBACChain(ONLY_ROLE_0_REG_JSON, network, subnet)
             # RBAC registration chain that contains both Role 0 -> Role 3 (proposer)
-            case RoleID.PROPOSER:
+            case Chain.Role0_With_Proposer:
+                return RBACChain(ROLE_3_REG_JSON, network, subnet)
+            # RBAC registration chain that contains all known roles
+            case Chain.All:
```

Comment:
I think this ALL is a bit confuse because it is the same as above.

Reply:
I would leave it, because at this moment it is fully the same as previous one, but after we will bring new roles into our system it will be not longer the same.